### ISSUE TITLE
Doesn't fix what TooltipFix fixes

### ISSUE THREAD

**Issue by itsdinkd** [isXander/AdaptiveTooltips#19]
Note the cutoff in screenshot

![image](https://user-images.githubusercontent.com/35115304/222929795-a339e7b1-598e-4da4-b5ca-517a5c9b49fd.png)

**Comment by itsdinkd** [comment]
when i switch fromt the default, Screen width + bedrock center to remaining_with and bedrock centering off, this happens.

![image](https://user-images.githubusercontent.com/35115304/222932463-6474ffbd-1157-47c3-bc09-e3c543619bc0.png)

**Comment by isXander** [comment]
are you sure you are configuring it correctly? There is a config menu.

**Comment by itsdinkd** [comment]
> are you sure you are configuring it correctly? There is a config menu.

Yes. i configured it correctly. this mod is no different in the sense of configuration than all the other mods I configured in my pack. 
https://pastebin.com/EFKq9eer

**Comment by isXander** [comment]
It may be a mod conflict. Can you send a mod list?

**Comment by Garwinium** [comment]
> are you sure you are configuring it correctly? There is a config menu.

sorry to ask randomly, but where is this config menu? ithere isnt a config button in mod menu like with most mods

**Comment by Garwinium** [comment]
im wondering because I would like to configure the mod as it says it is highly configurable, and my tooltips still go off the screen so i wanted to see if any of the config options would fix it

**Comment by isXander** [comment]
Stale
